### ISSUE TITLE
Cartes postales anciennes 

### ISSUE THREAD

**Issue by orovellotti** [PnX-SI/GeoPaysages#182]
On peut aussi imaginer de demander aux utilisateurs d'uploader leurs cartes postales anciennes
 
![image](https://github.com/PnX-SI/GeoPaysages/assets/1989428/8dc071bf-6dc9-4c77-a6e5-b73d118e631a)

**Comment by geobrun** [comment]
De notre côté, on serait intéressé par une fonctionnalité permettant de fournir des cartes postales anciennes non localisées sur le site internet afin que des personnes puissent les géolocaliser approximativement

**Comment by orovellotti** [comment]
Oui intéressant on peut en discuter ?

**Comment by camillemonchicourt** [comment]
On peut déjà charger des cartes postales géolocalisées. 
C'est surtout intéressant si on a refait des prises de vue du même point de vue plus tard pour comparer l'évolution qui est l'objectif central de GeoPaysages.

**Comment by orovellotti** [comment]
L'enjeu est d'ouvrir la participation à plus de monde et surtout de faire connaitre ces observatoires

**Comment by xavyeah39** [comment]
Effectivement, le sujet des cartes postales anciennes souvent lié aux OPP revient souvent.
Il est déjà possible de distiguer ce type de prise de vue dans un observatoire spécifique avec l'ajout de la fonctionnalité "multi-observatoires" de la v1.3.0 à paraître.

Mais ajouter une dimension collaborative à GeoPaysages serait intéressante de manière générale.
Le besoin a déjà été exprimé et ça rejoint ton autre ticket #180.

Pour permettre aux visiteurs de soumettre des cartes postales anciennes, cela pourrait être assez "simple" avec l'app web dans la mesure où : 

- ces contributions ne se feront pas in-situe (à priori) mais plutôt depuis chez soi, tranquilement, après avoir scanné ou pris en photos la jolie carte postale retrouvée dans la vieille male de mamie.
- pas vraiment de contraintes de suivi d'une série puisque c'est une contribution nouvelle où l'on propose une carte ancienne (sauf si on considère qu'on peut avoir plusieurs millésimes de cartes postales anciennes d'un même point de vue et que ça devienne une "série"... pas forcément fréquent mais possible sur des sites "emblématiques")

Mais finalement, est-ce que ce n'est pas déjà ce qu'on pourrait faire avec GN Citizen et les programmes de type "sites" ? 
En tout cas techniquement, tout est là (mais c'est une autre app...)

**Comment by camillemonchicourt** [comment]
Je comprends pas bien.
Car comme j'indiquai plus haut, selon moi, les cartes postales anciennes sont intéressantes dans GeoPaysages si on refait des prises de vue du même endroit pour comparer les évolutions.

**Comment by geobrun** [comment]
D'accord avec Camille, c'est l'intérêt que l'on voyait de notre côté : notre idée, c'est que ce soit le gestionnaire du site internet qui propose des cartes postales anciennes non géolocalisées, que des habitants proposent ensuite une localisation et qu'on valide ensuite les cartes postales nouvellement localisées comme des nouveaux sites à reconduire.

**Comment by xavyeah39** [comment]
> les cartes postales anciennes sont intéressantes dans GeoPaysages si on refait des prises de vue du même endroit pour comparer les évolutions.

Oui oui bien sûr, il faut ensuite qu'il y a ait des reconductions photos des cartes postales transmises pour que ça ai du sens de les intégrer dans GeoPaysages.
Je me suis peut être mal exprimé mais je voulais dire que cette première contribution qui consisterai à uploader une carte postale (localisée ou non) impose moins de contraintes qu'une contribution de reconduction où l'on demanderai aux utilisateurs de reproduire un cliché "à l'identique" (cf. #180)

> notre idée, c'est que ce soit le gestionnaire du site internet qui propose des cartes postales anciennes non géolocalisées, que des habitants proposent ensuite une localisation et qu'on valide ensuite les cartes postales nouvellement localisées comme des nouveaux sites à reconduire.

Ok, je n'avais compris le besoin comme ça.
C'est vrai que c'est une autre approche si on ne demande pas aux gens de transmettre des cartes anciennes mais seulement de localiser une collection de cartes que l'on a déjà.
C'est intéressant aussi et complémentaire si on propose aussi aux utilisateurs de transmettre des cartes anciennes du territoires qui ne sont pas déjà dans la "collection" initiale (surtout si on a pas déjà capitalisé une collection de cartes postales ^^)

**Comment by camillemonchicourt** [comment]
Je pense que l'enjeu d'un tel objectif serait surtout de l'animation et de la communication, plus qu'un outil.
Si on communique bien et anime la démarche, un email pour recevoir les cartes postales peut déjà bien fonctionner.

**Comment by geobrun** [comment]
> Je pense que l'enjeu d'un tel objectif serait surtout de l'animation et de la communication, plus qu'un outil.
> Si on communique bien et anime la démarche, un email pour recevoir les cartes postales peut déjà bien fonctionner.

Oui, c'est comme ça que l'on fonctionne de notre côté pour le moment.

**Comment by orovellotti** [comment]
Effectivement la série n'apporte pas grand chose puisque les cartes postales ne sont pas ou très rarement rééditer avec une novelle image au même endroit.

J'aime bien l'idée de coupler avec un GN-Citizen pour décider ensuite si on en fait une série.

Je pense qu'un tel programme pourrait avoir du sucées avec une com groupée avec un journal local (la Marseillaise, la Provence, le dauphine etc ...)

J'ai pas mal de contacts avec ces journalistes.

**Comment by FrancoiseBD** [comment]
je rejoins mes collegues, pour simplifier le travail de modération, plus simple de porter une démarche d'animation avec un mail/contact précisé sur le site de l OPP qui permet au public d'envoyer photos ou cartes postales, et le gestionnaire decide ou non de publier (au risque de se retrouver envahi de photos redondantes ou hors sujet ou mal cadrées)
coté Luberon, on a des points de vue initiaux basés sur des cartes postales 1900, et on a fait la reconduction 
on a proposé un volet participatif via geocaching de ces points de vue, et les reconductions proposées sont sympas mais tres rarement cadrées...
